docs/docusaurus.config.js

```diff
@@ -21,6 +21,19 @@ module.exports = {
         },
       ],
     },
+    footer: {
+      links: [
+        {
+          title: "Questions or comments?",
+          items: [
+            {
+              label: "Let us know!",
+              to: "https://stellarform.typeform.com/to/r4FiNpX1",
+            },
+          ],
+        },
+      ],
+    },
```

Comment:
the latest minor version of docusaurus breaks if you don't have a footer, so I guess now we need footer links. Throwing in the typeform link, as it could be useful here